configs/projects/puppet-agent.rb

```diff
@@ -1,6 +1,8 @@
 project "puppet-agent" do |proj|
   platform = proj.get_platform
 
+  proj.conflicts "pe-r10k", "2.5.0"
```

Comment:
Would you mind adding a code comment for our future selves explaining why this conflicts is here? Also, maybe include the version of PE that this conflicts with in said comment?